Example/TABTestKit_ExampleUITests/OtherElementsTests.swift

```diff
@@ -153,6 +153,12 @@ final class OtherElementsTests: TABTestCase, SystemPreferencesContext {
       Then(the: value(of: otherElementsScreen.pageIndicator, is: "page 1 of 3"))
     }
     
+    Scenario("Seeing the image") {
+      Given(I: scroll(otherElementsScreen, .downwards, until: otherElementsScreen.picker, is: .visible))
+      Given(I: see(otherElementsScreen.image))
```

Comment:
This would read nicer as a Given, When, Then or a Given, Then, And - if you feel the second line is an assertion rather than an action.